docs/src/dev-guide/coding-guidelines.md

```
@@ -53,6 +53,181 @@ To avoid including a state variable in a React Hook's dependency array, you can
+
+## Store structure
+
+### Type definitions
+
+Split store types into three interfaces:
+
+* `{Name}Values` - state variables
+* `{Name}Actions` - action functions (methods that update state)
+* `{Name}State` - union of values and actions
+
+For large stores, split `{Name}State` into feature-specific slices (`{Name}{Feature}Slice`) and 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)